docs/09-troubleshooting/common-issues.mdx

```diff
@@ -2,8 +2,93 @@
 
 Below you'll find a collection of answers to commonly reported problems.
 
+## Unity Activation
```

Comment:
The diff here might be harder to follow, but I moved issues related to activation on top as they are the most common ones so far. Then I added some new sections. 👍